Python/Tests/TestAdapterTests/TestDiscovererTests.cs

```diff
@@ -65,6 +66,37 @@ public abstract partial class TestDiscovererTests {
             DiscoverTests(testEnv, new[] { testFilePath }, runSettings, expectedTests);
         }
 
+        [TestMethod, Priority(0)]
+        [TestCategory("1s")]
+        public void DiscoverPytestTimeoutError() {
+            var testEnv = TestEnvironment.Create(Version, FrameworkPytest);
+
+            var testFilePath = Path.Combine(testEnv.SourceFolderPath, "test_timeout_pt.py");
+            File.Copy(TestData.GetPath("TestData", "TestDiscoverer", "Timeout", "test_timeout_pt.py"), testFilePath);
+
+            var waitTimeInSeconds = "1";
```

Comment:
new timeout test.. sets discovery timeout to 1 second then checks the errors printed